debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresType.java

```diff
@@ -19,25 +19,29 @@
  */
 public class PostgresType {
 
-    public static final PostgresType UNKNOWN = new PostgresType("unknown", -1, Integer.MIN_VALUE, null);
+    public static final PostgresType UNKNOWN = new PostgresType("unknown", -1, Integer.MIN_VALUE, null, null, null);
 
     private final String name;
     private final int oid;
     private final int jdbcId;
+    private final PostgresType baseType;
     private final PostgresType elementType;
     private final TypeInfo typeInfo;
+    private final int modifiers;
 
-    public PostgresType(String name, int oid, int jdbcId, TypeInfo typeInfo) {
-        this(name, oid, jdbcId, typeInfo, null);
+    public PostgresType(String name, int oid, int jdbcId, TypeInfo typeInfo, PostgresType baseType, PostgresType elementType) {
+        this(name, oid, jdbcId, TypeRegistry.NO_TYPE_MODIFIER, typeInfo, baseType, elementType);
     }
 
-    public PostgresType(String name, int oid, int jdbcId, TypeInfo typeInfo, PostgresType elementType) {
+    public PostgresType(String name, int oid, int jdbcId, int modifiers, TypeInfo typeInfo, PostgresType baseType, PostgresType elementType) {
```

Comment:
elementType is an exception, can we have another constructor without theis parameter?